backend/src/endpoints/project.ts

```diff
@@ -1,9 +1,12 @@
-import { getCollections, validateProject } from "../services/dbservice";
+import { getCollection, validate } from "../services/dbservice";
 import { Request, Response } from "express";
 import { ObjectId } from "mongodb";
+import { Project } from "../models/Project";
 
-export const getAllProjects = async (_req: Request, res: Response) => {
-    const { projects } = getCollections();
+export const getAllProjects = async (req: Request, res: Response) => {
+    console.log(req.cookies.AlgoToken); // if set user can be obtained from this token and used to filter projects
```

Comment:
Wydaje mi się, że console.log to pozostałość po debugowaniu, chyba nie jest tu konieczny

Reply:
ten console.log to jest do zadania autoryzacja żeby pokazać że tam można dostać AlgoToken ale mogę go dać w komentarz